tests/installation/test_chooser.py

```diff
@@ -8,6 +8,7 @@
 from packaging.tags import Tag
 from poetry.core.packages.package import Package
 
+from poetry.console.exceptions import PoetryRuntimeError
```

Comment:
Shouldn't changing `test_chooser` be in the same commit as changing `chooser`?

Reply:
Yup.